tests/unit/test_charm_2.py

```diff
@@ -1,12 +1,48 @@
+import os
 import unittest
 import unittest.mock
 
 import ops.testing
 from charm import WordpressCharm
 
+METADATA_FILE = os.path.join(os.path.split(__file__)[0], "..", "..", "metadata.yaml")
+
 
 class TestWordpressK8s(unittest.TestCase):
     def setUp(self):
+        container_fs = {}
```

Comment:
Is this variable necessary? Could this and the below line be combined?

Reply:
Just don't want to access `self` variable in function closure, I guess it works, but it will look confusing.

Reply:
Oh I didn't notice that the functions below also make use of it, never mind